agot-bg-game-server/src/client/GameLogListComponent.tsx

```diff
@@ -454,6 +454,13 @@ class GameLogListComponent extends Component<GameLogListComponentProps> {
                         ))}
                     </ul>
                 </>;
+            case "enemy-port-taken":
+                house = this.game.houses.get(data.house);
+                return <>
+                    <p>
+                        <strong>{house.name}</strong> has taken {data.shipCount} ship{data.shipCount==1?"":"s"} from {data.port}.
```

Comment:
Adding the `affectedHouse` in the game log and the number of ships that were destroyed would be nice.

Also, the phrasing could mention that the ships were taken _because_ the port changed controller. Idk what sentence could translate that best.